apps/web/components/ui/ImageText/ImageTextForm.vue

```diff
@@ -270,50 +262,65 @@
 
         <div class="py-2">
           <div class="flex justify-between mb-2">
-            <UiFormLabel>Button Link</UiFormLabel>
+            <UiFormLabel>Button Label</UiFormLabel>
```

Comment:
```suggestion
            <UiFormLabel>Link target</UiFormLabel>
```
Label is the input above, right?